### PR TITLE
[10.0] OFX: fix start/end balance

### PR DESCRIPTION
**Pull request by alexis-via**
Remove start/end dates in name of statement, because it's better to have it via 2 computed fields

### CODE REVIEW

**Comment by alexis-via** [comment]
The fix on start/end balance is not new ; it restores the code before this commit:
https://github.com/OCA/bank-statement-import/commit/c7c6954df7d1932afe539a17c855cb87acec70b3

File: account_bank_statement_import_ofx/wizard/account_bank_statement_import.py [hunk 1]
```diff
@@ -65,30 +65,23 @@ def _parse_file(self, data_file):
...
         except Exception, e:
             raise UserError(_(
                 "The following problem occurred during import. "
                 "The file might not be valid.\n\n %s") % e.message)
 
+        balance = float(ofx.account.statement.balance)
         vals_bank_statement = {
-            'name': _('Account %s %s > %s') % (
-                ofx.account.number, start_date, end_date),
+            'name': ofx.account.number,
```
Comment (StefanRijnhart) [inline account_bank_statement_import_ofx/wizard/account_bank_statement_import.py:81]: Over at the CAMT corner, we are really struggling with statement identifiers that are recycled every year. Are you sure that is not the case for OFX?

**Review by StefanRijnhart — APPROVED** [review]
Just a question

**Review by pedrobaeza — APPROVED** [review]
@alexis-via please answer later the question from Stefan, although I merge as it's not related.

**Pull request merged**